test/torch/tensors/test_poly.py

```diff
@@ -0,0 +1,60 @@
+import syft 
+from syft.frameworks.torch.tensors.Polynomial import PolynomialTensor
+
+import math
+
+class TestPolynomialTensor:
+
+    """ Test cases to ensure working of Polynomial Tensor. The tests under these ensure that the error between actual funcion values and approximations do not deviate too much"""
+
+    def __init__(self):
+
+        self.tensor = PolynomialTensor()
+        # Maximum permissible error as calculated by EvalRelative under PolynomialTensor
+        self.threshold = 0.1
+
+    def EvalRelative_Test(self, x_true, x_pred):
+
+        assert self.tensor.EvalRelative(15 / 7.5) == 0.5
+        assert self.tensor.EvalRelative(15 / 1.5) == 0.9
+        assert self.tensor.EvalRelative(15 / 15) == 0
+
+    def SigmoidTest(self):
+
+        test_range = 4
+        for i in range(-test_range, test_range, 1):
```

Comment:
It might give clearer error messages if we parameterize these rather than putting a for loop in the test